yarn.lock

```diff
@@ -334,6 +339,15 @@
     eslint "^5.0.1"
     eslint-plugin-eslint-comments "^3.0.0"
 
+"@idearium/log@1.0.0-beta.2":
```

Comment:
@allanchau, this doesn't seem right? It should be `beta.6`?

Reply:
@smebberson I've updated it now, but it shouldn't really matter because it isn't used? We could probably add it to gitignore.

Reply:
What do you mean? It's totally used? See https://github.com/idearium/idearium-lib/pull/90/files#diff-6bbe3edcb5cef2c785a41583b1977e9b5487596b908cefcb6b1a13696b77664aL4? Am I missing something?

Reply:
@smebberson I mean the yarn.lock file in the root of the monorepo isn't deployed with the packages? It shouldn't affect how it is used? I think as long as the package.json in the packages is correct it should be ok